android/linting/src/main/java/org/smartregister/lint/FhirEngineCreateUpdateMethodCallRule.kt

```diff
@@ -0,0 +1,59 @@
+package org.smartregister.lint
+
+/**
+ * Created by Ephraim Kigamba - nek.eam@gmail.com on 14-06-2023.
+ */
+import com.pinterest.ktlint.rule.engine.core.api.ElementType
+import com.pinterest.ktlint.rule.engine.core.api.Rule
+import com.pinterest.ktlint.rule.engine.core.api.RuleId
+import com.pinterest.ktlint.rule.engine.core.api.parent
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+
+class FhirEngineCreateUpdateMethodCallRule : Rule(
+    ruleId = RuleId("$CUSTOM_RULE_SET_ID:fhir-engine-create-update-used"),
+    about =
+    About(
+        maintainer = "Ephraim Kigamba"
```

Comment:
@ndegwamartin who should be noted as the maintainer for this?

Reply:
I think we can have **OpenSRP Developers** or **OpenSRP Team**

https://github.com/orgs/opensrp/teams/opensrp-developers